scripts/qa_crowbarsetup.sh

```diff
@@ -2032,12 +2032,11 @@ function onadmin_proposal()
     if iscloudver 5plus; then
         update_one_proposal dns default
     fi
-    if [ "$networkingplugin" = "vmware" ] && iscloudver 5plus ; then
```

Comment:
@aspiers just saw your comment in https://github.com/SUSE-Cloud/automation/commit/5b2827557ba42222d77296744ee34529d33480b8
wasn't referenced here in this PR